pkg/loop/telem.go

```diff
@@ -57,18 +57,69 @@ func NewGRPCOpts(registerer prometheus.Registerer) GRPCOpts {
 
 // SetupTracing initializes open telemetry with the provided config.
 // It sets the global trace provider and opens a connection to the configured collector.
-func SetupTracing(config TracingConfig) (err error) {
+func SetupTracing(config TracingConfig) error {
```

Comment:
This method is preserved since it is exported.

Reply:
There are not that many usages of it thought, see https://github.com/search?q=%22loop.SetupTracing%22&type=code
So it should be feasible if we decide to deprecate, move or change `SetupTracing`